cmd/frontend/internal/httpapi/completions/handler.go

```diff
@@ -577,10 +574,14 @@ func checkClientCodyIgnoreCompatibility(ctx context.Context, db database.DB, r *
 			cvc.constraint = ">= 5.5.8-0"
 		}
 	default:
-		return &codyIgnoreCompatibilityError{
-			reason:     fmt.Sprintf("please use one of the supported clients: %s, %s, %s.", types.CodyClientVscode, types.CodyClientJetbrains, types.CodyClientWeb),
-			statusCode: http.StatusNotAcceptable,
-		}
+		// By default, allow requests from any client. We only want to reject
+		// requests from older client versions that we know definitily don't
+		// support context filters. A malicious client can always bypass this
+		// check anyways by faking to be "jetbrains". All agent-based clients
+		// (JetBrains, Eclipse, Visual Studio) support context filters out of
+		// the box since the original support was added for JetBrains GA in May
+		// 2024.
```

Comment:
```suggestion
		// By default, allow requests from any client. We only want to reject
		// requests from older client versions that we know definitely don't
		// support context filters.
		// All agent-based clients (JetBrains, Eclipse, Visual Studio) support 
		// context filters out of the box since the original support was added 
		// for JetBrains GA in May 2024.
```

Just feel like we don't want to advertise this!